src/logging.h

```diff
@@ -132,6 +132,13 @@ namespace BCLog {
         bool DisableCategory(const std::string& str);
 
         bool WillLogCategory(LogFlags category) const;
+        /** Returns a vector of the log categories */
+        std::vector<LogCategory> LogCategoriesList();
+        /** Returns a string with the log categories */
+        std::string LogCategoriesString()
```

Comment:
Moving the implementation to `logging.cpp` could avoid adding the compile-time dependency on `util/string.h` here.

Reply:
`util/string.h` is relatively trivial to parse for a compiler, compared to `serialize.h` or the boost `fs.h`, so I thought it could be ok to include it here.

Happy to change though. You gave it an ACK, so let me know if I should invalidate that.